newscoop/src/Newscoop/NewscoopBundle/Resources/views/Plugins/index.html.twig

```diff
@@ -140,6 +141,14 @@ li:after {content: none;}
                     </form>
                     <!-- class="form-inline" -->
                 </fieldset>
+                {% else %}
+                <div class="uploaded-plugins" style="margin-bottom: 50px; padding: 10px 10px 30px 10px; border: 1px solid #ccc; background-color: #FFF;">
+                <div class="alert alert-danger" style="font-size: 14px;" role="alert">{{ 'newscoop.plugins_manager.private_plugins_not_writable'|trans }}</div>
```

Comment:
Maybe this could be:

{{ privatePluginsPath }} {{ 'newscoop.plugins_manager.private_plugins_not_writable'|trans }}

where private_plugins_not_writable is the following string:

 is not writable by your web server.